packages/contracts-ethers/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@aragon/core-contracts-ethers",
+  "version": "0.0.1",
+  "description": "Aragon V2 DAO Framework. Contract definition for Ethers.js",
+  "main": "dist/bundle-cjs.js",
+  "module": "dist/bundle-esm.js",
+  "types": "dist/bundle.d.ts",
+  "scripts": {
+    "build:contracts": "cd ../contracts && yarn build",
+    "build": "find ../contracts/artifacts/contracts/ -name '*.json' -type f | grep -v '.dbg.json' | xargs typechain --target=ethers-v5 --out-dir ./types",
+    "build:npm": "rollup --config rollup.config.ts",
+    "prepublishOnly": "yarn build:contracts && yarn build && yarn build:npm"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/aragon/core.git"
+  },
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/aragon/core/issues"
+  },
+  "homepage": "https://github.com/aragon/core#readme",
+  "devDependencies": {
+    "@rollup/plugin-typescript": "^8.3.1",
+    "@typechain/ethers-v5": "^10.0.0",
+    "rollup": "^2.70.1",
+    "rollup-plugin-dts": "^4.2.0",
+    "ts-node": "^10.7.0",
```

Comment:
Be careful, @nivida found the reason for the `Error HH13: Your Hardhat project uses typescript, but ts-node is not installed.` error. The version of `ts-node` that we are using has to match the version that hardhat uses internally.

Reply:
It should be 8.1.0 (see commit 51896b5348eb248e472250720d39eb1f8f81a309).

Reply:
But @Michael-A-Heuer the fact that `contracts-ethers` uses another version of `ts-node` should not be having any impact on the sibling subpackage (contracts + hardhat). I have updated the `contracts/package.json` entry for `ts-node`